lib/components/atoms/Icon/Icon.style.js

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components';
+
+export default css`
```

Comment:
I am not happy with this solution, but for now this will do. 

Let's plan to fix it soon. @jainaman23 assign a ticket to your name